frontends/mit-open/src/page-components/ManageListDialogs/ManageListDialogs.tsx

```diff
@@ -186,7 +145,6 @@ const UpsertLearningPathDialog = NiceModal.create(
         />
         <TextField
           required
-          className="form-row"
           label="Description"
           name="description"
           placeholder="List Description"
```

Comment:
@steven-hatch Do we still want the description field? Can always remove it later if not sure.